GiraffeApp/Program.fs

```diff
@@ -35,27 +36,33 @@ let pageHandler (title: string)(view: XmlNode list)(next: HttpFunc)(ctx: HttpCon
     htmlNodes nodes next ctx
     
 [<CLIMutable>]
-type CountPayload = { Count: int }
+type IncrementForm = { Count: int }
 
-let incrementHandler: HttpHandler =
-    let culture = CultureInfo.CreateSpecificCulture("en-US")
-    bindQuery<CountPayload> (Some culture) (fun payload -> 
-        htmlNodes (Views.counter payload.Count)
-    )
+let incrementHandler(next: HttpFunc)(ctx: HttpContext): HttpFuncResult =
+    let af = ctx.GetService<IAntiforgery>()

```

Comment:
We can remove all Csrf code since it will be implemented in requiresCsrfToken. 
The incrementHandler will contain only its logic.


```suggestion
    bindForm<IncrementForm> None (fun payload -> 
        htmlNodes (Views.counter payload.Count (af.GetAndStoreTokens ctx))) next ctx
```